### PR TITLE
Updating Aditya's Info

### PR DESCRIPTION
**Pull request by aditya-uw**
1st commit so far I have added one line in the News section. Not sure what links to add. I will eventually be on a page with the other scholars like [here](https://expd.uw.edu/mge/scholar-profiles/). But for now, this is all I have to put.

I also plan to revisit my bio tomorrow to see if I want to update any information there so I will keep this PR open a bit for that.

### CODE REVIEW

**Comment by aditya-uw** [comment]
Okay @leewujung I believe I am satisfied with my updates.

- I added a blurb about how we collect our data in my profile and also made some updates about concentrations and interests.
- Added MGRS info in the "News" section along with a link to the standard Mary Gates Scholarship page. 
- Updated my photo: I tried resampling to 270x270 before uploading so it doesn't become unexpectedly blurry.
- Also fixed a typo I caught in the "News" section in the first entry: changed "boiological" to "biological"

Take a look and let me know if you have suggestions!

File: content/authors/adkris/_index.md [hunk 1]
```diff
@@ -66,11 +65,11 @@ user_groups:
 - Researchers
 ---
 
-I am a student at the University of Washington majoring in Electrical Engineering with a concentration in Digital Signal Processing. I am expected to graduate the Spring of 2024 and am interested in pursuing graduate research after my bachelor's degree.
+I am a student at the University of Washington majoring in Electrical Engineering and pursuing Digital Signal Processing and Neural Engineering concentrations. I am expected to graduate the Spring of 2024 and am interested in pursuing graduate research after my bachelor's degree.
 
-My core competency is in programming, with an expertise in Java and Python. I aim to use my programming skills to aid in ecological & environmental conservation. 
+My core competency is in programming, with an expertise in Python and Java. I am currently studying how signal processing methods are used in passive acoustic monitoring projects to address project-specific goals and aid in ecological conservation.
```
Comment (leewujung) [inline content/authors/adkris/_index.md:70]:
```suggestion
My core competency is in programming, with an expertise in Python and Java. I am currently applying signal processing techniques in passive acoustic monitoring of bats with a goal toward (?) ecological conservation.
```
I shortened the sentence a bit, but not sure about the goal part, hence the "?"

File: content/authors/adkris/_index.md [hunk 2]
```diff
@@ -66,11 +65,11 @@ user_groups:
...
 
-At Echospace, I am currently working on using machine learning methods on echolocation data to understand the foraging and social behaviors of bats in the wild. 
+At Echospace, I am developing and applying computational methods on echolocation data to understand the foraging and social behaviors of bats in the wild. We collect our echolocation data from local bat colonies by deploying AudioMoths, which are autonomous recording units, at the Union Bay Natural Area, beside the Husky Football Stadium.
```
Comment (leewujung) [inline content/authors/adkris/_index.md:72]:
```suggestion
At Echospace, I am developing and applying computational methods on echolocation data to understand the foraging and social behaviors of bats in the wild. We collect our echolocation data from local bat colonies by deploying autonomous recording units (AudioMoths [ADD LINK]), at the Union Bay Natural Area [ADD LINK] right bu the UW Husky Football Stadium.
```
The ADD LINK parts are suggestions, you can decide if you actually want them hyperlinked.

File: content/newslist.dat [hunk 3]
```diff
@@ -1,3 +1,5 @@
+**[12/16/2022]** Aditya has been awarded the [Mary Gates Research Scholarship](https://expd.uw.edu/mge/apply/research/)!
```
Comment (leewujung) [inline content/newslist.dat:1]: Maybe add something at the end about monitoring bats in UBNA?

**Comment by leewujung** [comment]
@aditya-uw : Good job in updating the info! I added some comments. 
For the picture, I am not sure if upsampling actually helps with resolution? and 10 MB is a bit huge, haha. I wonder if making the head bigger would be good for how it looks on the site. Otherwise, feel free to merge once you are happy with everything.

File: content/authors/adkris/_index.md (hunk 1 above)
Comment (aditya-uw) [reply content/authors/adkris/_index.md]: Honestly, I'll keep it as "toward ecological conservation" but the more I think about it, I am starting to see bats as a very interesting system to study on multiple levels (echolocation, bioindicators, social dynamics) so my goal feels really to just study bats haha.

File: content/authors/adkris/_index.md (hunk 2 above)
Comment (aditya-uw) [reply content/authors/adkris/_index.md]: Added a link to the Audiomoth page! Good suggestion!

**Comment by aditya-uw** [comment]
@leewujung: you were right that it was unusually big haha! I was doing 270x270 inches and not 270x270 pixels. 

I think I'm ready to merge now!

**Comment by aditya-uw** [comment]
@leewujung: You were right about the image being unusually large, I had set it resized to 270 inches not 270 pixels haha.

I'll self-merge now!

**Comment by aditya-uw** [comment]
Oh wait I don't have write access to merge PRs in this repositiory

**Comment by leewujung** [comment]
Oh haha, no problem I can merge it. :)

**Pull request merged**